plugins/techdocs-backend/examples/documented-component/documented-component.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: documented-component
+  description: A Service with TechDocs documentation
+  annotations:
+    spotify.com/techdocs-ref: 'dir:./'
```

Comment:
Do we really need Spotify here? TechDocs will be a core feature of Backstage and should therefore not have to be branded.

Reply:
I got the impression we were using that prefix to underline it's core stuff. Is the conventions documented somewhere?

Reply:
Maybe `backstage.io` is better?